crates/pathfinder/src/rpc.rs

```diff
@@ -1537,6 +1538,13 @@ mod tests {
                     .unwrap();
                 // Only asserting the hash because translating from Sequencer receipt to RPC receipt is pita.
                 assert_eq!(receipt.hash(), expected.transaction_hash);
+                assert_matches!(
+                    receipt,
+                    // FIXME: this should test an invoke instead
```

Comment:
And this one should be updated too after the test data has been fixed -- in that case we can actually assert on something meaningful inside the receipt.